client/my-sites/site-settings/traffic/main.jsx

```diff
@@ -33,6 +37,15 @@ const SiteSettingsTraffic = ( {
 		<SiteSettingsNavigation site={ site } section="traffic" />
 
 		<SeoSettingsHelpCard />
+		{ jetpackSettingsUiSupported &&
```

Comment:
Since this setting panel should only ever be used on Jetpack sites - should this check for `jetpackSettingsUiSupported` be moved to the component render itself?